src-doc/contenuti/requisiti.tex

```diff
@@ -0,0 +1,160 @@
+\chapter{Requsiti}
+
+\section{Introduzione}
+Di seguito sono riportati i requisiti che il gruppo \textit{SWEasabi} ha classificato utilizzando la metodologia definita nelle \textit{Norme di Progetto}.
+
+\section{Requisiti funzionali}
+
+\begin{center}
+    \begin{xltabular}{\linewidth}{|l|l|X|l|}
+        \hline
+        %Colora la riga di un grigio chiaro più chiaro
+        \rowcolor[gray]{0.9}
+        \textbf{Codice} & \textbf{Classificazione} & \textbf{Descrizione} & \textbf{Fonti} \\
+        \hline
+        RF\_01 & Obbligatorio & L'utente deve visualizzare un messaggio d'errore esplicativo in caso di errore di autenticazione & UC01 \\
```

Comment:
* Riformulato in modo che il soggetto sia il sistema.
* Ha senso specificare il sottocaso?
```suggestion
        RF\_01 & Obbligatorio & Il sistema deve mostrare un messaggio d'errore esplicativo all'utente in caso di errore di autenticazione & UC01.1 \\
```

Reply:
Credo abbia senso (e mi sembra pratica diffusa guardando in giro), però possiamo sicuramente discuterne. Concordo sulla correzione